qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -5,39 +5,54 @@
 
 package org.opensearch.knn.bwc;
 
-import static org.opensearch.knn.TestUtils.*;
+import java.io.IOException;
+
+import static org.opensearch.knn.TestUtils.NODES_BWC_CLUSTER;
 
 public class IndexingIT extends AbstractRollingUpgradeTestCase {
     private static final String TEST_FIELD = "test-field";
     private static final int DIMENSIONS = 5;
     private static final int K = 5;
-    private static final int ADD_DOCS_CNT = 10;
+    private static final int NUM_DOCS = 10;
 
-    public void testKnnDefaultIndexSettings() throws Exception {
+    public void testKNNDefaultIndexSettings() throws Exception {
         waitForClusterHealthGreen(NODES_BWC_CLUSTER);
         switch (getClusterType()) {
             case OLD:
                 createKnnIndex(testIndex, getKNNDefaultIndexSettings(), createKnnIndexMapping(TEST_FIELD, DIMENSIONS));
-                addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, 0, ADD_DOCS_CNT);
+                int docIdOld = 0;
+                addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, docIdOld, NUM_DOCS);
                 break;
             case MIXED:
                 if (isFirstMixedRound()) {
-                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, 10, K);
-                    addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, 10, ADD_DOCS_CNT);
-                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, 20, K);
+                    int totalDocsCountFirstMixedRound = NUM_DOCS;
+                    int docIdFirstMixedRound = NUM_DOCS;
+                    validateKNNIndexingOnUpgrade(totalDocsCountFirstMixedRound, docIdFirstMixedRound);
```

Comment:
See, we can hav 29 and 33 be the same variable and remove validateKNNIndexingOnUpgrade from the if/else block. No reason to call it docIdFirstMixedRound and docIdSecondMixedRound. Just have one variable

Reply:
ack